### ISSUE TITLE
authentication with sso

### ISSUE THREAD

**Issue by laurentiu2** [uvdesk/community-skeleton#250]
Hi,
is there any possibility to login other than user/password? (SSO, ID Connect, ecc)

My use case: I use different apps that allows a single point of authentication and I need uvdesk community to be the same. Is it possible? Does it have this feature?

Thanks,

**Comment by papnoisanjeev** [comment]
@laurentiu2 

No, For now we have not added this feature in UVDesk Opensource.
but will try to add this feature in incoming releases.

**Comment by wibed** [comment]
has this changed?

https://www.uvdesk.com/en/blog/openid-connect-extending-oauth-based-authentication/
implies there is oidc, does this apply to the open source version aswell?

**Comment by Ludogo** [comment]
Hello 
we need open ID connect feature too. any update on this point ?

**Comment by olsn** [comment]
If you go to https://www.uvdesk.com/en/feature-compare/ there (under security) it is listed as a non-opensource-feature.

**Comment by papnoisanjeev** [comment]
@olsn 

Yes, For Opensource sso not available.

**Comment by althaus** [comment]
Any plans on moving this at some point into the free plan? It's an elemental feature in my eyes and I'd tend to fork or ditch uvdesk otherwise. :(

**Comment by celi28** [comment]
Hello,  
Still, no plan to move this feature to the OpenSource version?

Almost all business companies can't deploy this project due to this blocking point...

**Comment by nmbgeek** [comment]
3 years later and a very basic feature still is not included.

> @laurentiu2
> 
> No, For now we have not added this feature in UVDesk Opensource. but will try to add this feature in incoming releases.

**Comment by manoonabbasi** [comment]
Hi everyone,

I've successfully implemented Google OAuth2 / GSuite login with UVdesk, enabling seamless SSO for agents. If you're interested in setting this up or need guidance, feel free to reach out. I’ll also be uploading the bundle soon for those who want to integrate it directly!

Best,
Manoons

**Comment by papnoisanjeev** [comment]
@manoonabbasi 

Thanks for you efforts for this feature implementation.
You can create a pull request for the same so that team will review it and if everything goes good we can add it into main repository, so that community can use it.
